mlflow/store/file_store.py

```diff
@@ -207,7 +207,8 @@ def create_run(self, experiment_id, user_id, run_name, source_type,
         run_uuid = uuid.uuid4().hex
         artifact_uri = self._get_artifact_dir(experiment_id, run_uuid)
         num_runs = len(self._list_run_uuids(experiment_id))
-        run_info = RunInfo(run_uuid=run_uuid, experiment_id=experiment_id, name="Run %s" % num_runs,
+        run_info = RunInfo(run_uuid=run_uuid, experiment_id=experiment_id,
```

Comment:
+1 thanks for making this change

Reply:
If you want me to deliver just this piece of code, I can open another PR. However, I think that the fix to set default start_time above can also go in while issue #354 continues to track other fields. Please let me know.

Reply:
Thanks @adrian555, yeah would be great to have a separate PR for the run name fix - appreciate your hard work on these fixes!

Reply:
Thanks, opened https://github.com/mlflow/mlflow/pull/358.